src/client/Microsoft.Identity.Client/OAuth2/OAuth2Client.cs

```diff
@@ -80,6 +80,8 @@ internal async Task<MsalTokenResponse> GetTokenAsync(Uri endPoint, RequestContex
 
         internal async Task<T> ExecuteRequestAsync<T>(Uri endPoint, HttpMethod method, RequestContext requestContext, bool expectErrorsOn200OK = false, bool addCommonHeaders = true)
         {
+            var logger = requestContext.Logger;
```

Comment:
do you need the variable? you only use it once or use it later 215, if it's the same method .